src/pykeen/models/cli/builders.py

```diff
@@ -142,6 +142,15 @@ def main(
         )
         from ...pipeline import pipeline
 
+        if mlflow_tracking_uri:
```

Comment:
Can we generalize this part?
We should consider that we could also pass `experiment_id`, and `experiment_name` for the MLFlow tracker.

Reply:
Yes, but that should be a different PR. This will require writing new code to introspect on each model and programatically add options, like I did with models

Reply:
Ok 👍